### ISSUE TITLE
Add SEO and change title, description on the website

### ISSUE THREAD

**Issue by TheJoin95** [Schroedinger-Hat/peoplez#70]

**Comment by TheJoin95** [comment]
@Readpato this could be a good task for you

**Comment by Readpato** [comment]
Good idea! Since I'm not familiar to the Next.js ecosystem, do you know if there's already something predefined or a standardized way of modifying `<head>` properties in Next.js? I have only found this on the [documentation](https://nextjs.org/docs/pages/api-reference/components/head)

Normally in the Vue Ecosystem we utilize this agnostic [Unjs](https://unjs.io/) package [Unhead](https://unhead.unjs.io/) which allows you modify it super easily. Maybe if there isn't one we could adopt it? It's [pretty straightforward to implement it](https://unhead.unjs.io/setup/unhead/installation)

**Comment by TheJoin95** [comment]
You can take osday as reference. There is no need to install another dependency here as it'll be an easy and static head
